spec/spec_helper_local.rb

```diff
@@ -48,10 +48,24 @@ def param(type, title, param)
   it { is_expected.not_to eq(nil) }
 
   it {
-    is_expected.to run.with_params('foo', 'bar').and_return('md596948aad3fcae80c08a35c9b5958cd89')
+    is_expected.to run.with_params('foo', 'bar').and_return(
+      'md596948aad3fcae80c08a35c9b5958cd89'
+    )
   }
   it {
-    is_expected.to run.with_params('foo', 1234).and_return('md539a0e1b308278a8de5e007cd1f795920')
+    is_expected.to run.with_params('foo', 1234).and_return(
+      'md539a0e1b308278a8de5e007cd1f795920'
+    )
+  }
+  it {
+    is_expected.to run.with_params('foo', 'bar', nil, 'scram-sha-256').and_return(
+      'SCRAM-SHA-256$4096:YmFy$y1VOaTvvs4V3OECvMzre9FtgCZClGuBLVE6sNPsTKbs=:HwFqmSKbihSyHMqkhufOy++cWCFIoTRSg8y6YgeALzE='
```

Comment:
On my local system I'm getting a different value:
expected postgresql_password("foo", "bar", false, "scram-sha-256") to have returned "SCRAM-SHA-256$4096:YmFy$y1VOaTvvs4V3OECvMzre9FtgCZClGuBLVE6sNPsTKbs=:HwFqmSKbihSyHMqkhufOy++cWCFIoTRSg8y6YgeALzE=" instead of "SCRAM-SHA-256$4096:Zm9v$ea66ynZ8cS9Ty4ZkEYicwC72StsKLSwjcXIXKMgepTk=:dJYmOU6BMCaWkQOB3lrXH9OAF3lW2n3NJ26NO7Srq7U="
```